scripts/dd/dd-compare.py

```diff
@@ -0,0 +1,160 @@
+# Purpose: Takes in two PRISSMM data dictionaries and print out a simple comparison report
+# Author: Sage Bionetworks
+# Date: 29AUG2024
+
+import pandas as pd
+import synapseclient
+import argparse
+from io import StringIO
+
+# Log in 
+syn = synapseclient.Synapse()
+syn.login()
+
+# Set the default PRISSMM data dictionaries
+
+default1 = syn.get('syn52903784').path #v4.0.1
+default2 = syn.get('syn61600728').path #v4.1.4
+
+# Initialize arg parser
+parser = argparse.ArgumentParser(description="Compare two data dictionaries.")
+parser.add_argument('--df1', type=str, default=default1, help='File path to the older data dictionary CSV')
```

Comment:
If you set the default here to be synapse ids, then you can just do a syn.get at the bottom instead of what you're doing now. Then the argpaser doesnt rely on a syn.get

Reply:
Updated to only use syn.get at the bottom.